newsletter-november-2019.rst

```diff
@@ -0,0 +1,136 @@
+.. post:: November 04, 2019
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – November 2019
+#########################################
+
+`It's...<https://www.youtube.com/watch?v=ydyXFUmv6S4>`__ not Monty Python's Flying Circus. Sorry. It is, however, the November edition of the newsletter!
+
+The `Sydney conference </conf/australia/2019/>`__ is coming up in just a few short weeks. It'll be 14th-15th November, and there are still a handful of tickets left. So if you were on the fence, get them now!
+
+Looking forward to next year, the site for `Portland 2020 </conf/portland/2019/>`__ is live, as is the `Call For Proposals </conf/portland/2020/cfp>`__. We'd love to hear all your awesome talk ideas, so get working on them! The CFP closes on 10th January, 2020.
+
+Finally, we're changing policy for how we change policies, by introducing `Write the Docs Enhancement Proposals </blog/introducing-weps/>`__. The way we run the community has grown organically with the community itself, from seven whole years ago when it was just one conference . Our processes are only loosely defined - but that makes it harder for new people to understand the community and join the leadership. So we're trying WEPs, an idea taken from the open-source ecosystem. If you're interested, take a look at the `blog post </blog/introducing-weps/>`__, which explains it all much better.
```

Comment:
@ericholscher - can you tell me what you think of this as a summary? I'll probably try to edit it to be a little more concise tomorrow, but I'd like to check this reflects what you're trying to say

Reply:
Seems reasonable. I think it could be cleaned up a little bit, I'll take a swing at it:

```suggestion
Finally, Write the Docs as a community is formalizing how we change policies, by introducing `Write the Docs Enhancement Proposals </blog/introducing-weps/>`__. The way the community is run has grown organically, but there have been some growing pains as the community has expanded. Our processes are only loosely defined, which makes it harder for new people to understand the community and join the leadership. So we're trying WEPs, an idea taken from the open-source ecosystem. If you're interested, take a look at the `introduction blog post </blog/introducing-weps/>`__, which goes into more depth.
```

Reply:
missing "s" in positions. Otherwise :+1: to the change.